### PR TITLE
Add working tutorials from ROOT

### PR DESCRIPTION
**Pull request by shravan97**
Features in this PR : 
* Add support for 'Report' operation
* Add tutorials from ROOT project that are compatible with `PyRDF`
* Add a script to run tests and tutorials directly from the shell

Run instructions : 
1. `git clone https://github.com/shravan97/PyTDF.git PyRDF`
2. `cd /path/to/PyRDF/root/dir`
3. `git checkout tutorials`
4. `git rebase master`
5. `source run_tests.sh`

[Note :- Run the tutorials only **after** merging #23 . Preferably clone the repo after #23 is committed ]

### CODE REVIEW

File: PyRDF/Operation.py [hunk 1]
```diff
@@ -51,6 +51,7 @@ def _classify_operation(self, name):
         'Mean':ops.ACTION,
         'Sum':ops.ACTION,
         'Fill':ops.ACTION,
+        'Report':ops.ACTION
```
Comment (etejedor) [inline PyRDF/Operation.py:54]: This belongs to a different commit, if not to a different PR.

**Comment by etejedor** [comment]
Hi @shravan97 can you also add some means (e.g. script) for us to easily run the whole battery of tests and tutorials once we clone your repository?

File: PyRDF/Operation.py (hunk 1 above)
Comment (shravan97) [reply PyRDF/Operation.py:54]: @etejedor , sure. Will separate them :+1:

**Comment by shravan97** [comment]
@etejedor , sure. I'll add a script :+1:

File: PyRDF/Operation.py (hunk 1 above)
Comment (shravan97) [reply PyRDF/Operation.py:54]: @etejedor , I'm separating them into 2 different commits, because you'll need this commit (adding 'Report' action) to run the tutorials

**Comment by shravan97** [comment]
@etejedor , I've now added a shell script to run all tests and tutorials 👍

**Comment by shravan97** [comment]
And also @etejedor , shall I rename this repo now to PyRDF ?

**Comment by etejedor** [comment]
@shravan97 , yes, rename the repo to PyRDF please.
